docs/reference/configuration.rst

```diff
@@ -331,6 +335,28 @@ compatibility; on the platforms where it *is* used, the interpretation of the va
 platform specific. Refer to individual platform guides for details on how the provided
 value is interpreted.
 
+``package_path``
+~~~~~~~~~~~~~~~~
+
+The path to the root of a folder that will be packaged as an application. The contents
+of ``package_path`` is what will be shipped to the end user as the installed app.
+
+This is used to configure Briefcase to :doc:`package an external application
+</how-to/external-apps>`. It is not required if you are using Briefcase for the entire
+app creation process.
+
+If ``package_path`` is defined, ``sources`` must *not* be defined.
+
+``package_binary_path``
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
As discussed in external-apps.rst, this setting might not be necessary. But if it's kept, it should mention that it's only implemented on Windows.